src/Analysis/Ast/Impl/Analyzer/Handlers/FromImportHandler.cs

```diff
@@ -33,15 +33,14 @@ internal sealed partial class ImportHandler {
             }
 
             var rootNames = node.Root.Names;
-            IImportSearchResult imports = null;
             if (rootNames.Count == 1) {
                 var rootName = rootNames[0].Name;
                 if (rootName.EqualsOrdinal("__future__")) {
                     return false;
                 }
             }
 
-            imports = ModuleResolution.CurrentPathResolver.FindImports(Module.FilePath, node);
+            var imports = ModuleResolution.CurrentPathResolver.FindImports(Module.FilePath, node);
```

Comment:
Below this line it should be `mi.FullName == Module.Name`. And you can include this line into switch.

Reply:
OK